.github/scripts/downstream-project-spl-common.sh

```diff
@@ -27,3 +27,11 @@ fi
 
 # anza migration stopgap. can be removed when agave is fully recommended for public usage.
 sed -i 's/solana-geyser-plugin-interface/agave-geyser-plugin-interface/g' ./Cargo.toml
+
+# should be removed when spl bump their curve25519-dalek
+sed -i "s/^curve25519-dalek =.*/curve25519-dalek = \"4.1.3\"/" token/client/Cargo.toml
```

Comment:
```suggestion
```
The curve25519-dalek crate dependency in token-client is now removed with https://github.com/solana-labs/solana-program-library/pull/7179, so let's remove this line.